wix/main.wxs

```diff
@@ -16,7 +16,7 @@
         Manufacturer='Squidowl'
         Language='1033'
         Codepage='1252'
-        Version='0.0.{{ VERSION }}'>
+        Version='{{ VERSION }}.0.0'>
```

Comment:
I am not the big Windows user, but noticed that I have apps have different version notations:
Surely there must be a way for us to do yyyy.version like we do everywhere else? I just recall adding `0.0.{{ VERSION }}` because otherwise WiX would fail to compile 🤔 

![image](https://github.com/squidowl/halloy/assets/2248455/e0c7922e-5abc-47f7-bac5-ceca622c5511)

Reply:
Yea, well it is an Windows Installer SDK limitation (which is, IMHO, bad). That means that the Windows version will be always differ from the version in the VERSION file, as the major version must be less than 256. :-(

I will change the PR.

Reply:
To be more precise it seems a WiX 3 limitation. The Windows Installer SDK itself defines version as documented [here](https://learn.microsoft.com/en-us/windows/win32/msi/version). I made a brief attempt with WiX 5 and it seems that there is a warning if the version is "2004.7" and no error. Is it an option to rewrite the WiX description and migrate to WiX 5? I could make this.